.travis.yml

```diff
@@ -1,4 +1,3 @@
-sudo: false
```

Comment:
The `sudo:` key has been deprecated, see https://docs.travis-ci.com/user/reference/trusty/#container-based-infrastructure